test/e2e/framework/network_pods.go

```diff
@@ -198,7 +198,16 @@ func (np *NetworkPod) buildTCPCheckConnectorPod() {
 					Image: "busybox",
 					// We send the string 50 times to put more pressure on the TCP connection and avoid limited
 					// resource environments from not sending at least some data before timeout.
-					Command: []string{"sh", "-c", "for in in $(seq 50); do echo [dataplane] connector says $SEND_STRING; done | for i in $(seq $CONN_TRIES); do if nc -v $REMOTE_IP $REMOTE_PORT -w $CONN_TIMEOUT; then break; fi; done >/dev/termination-log 2>&1"},
+					Command: []string{"sh", "-c", ` function msg() {
+														for in in $(seq 50); do
+															echo [dataplane] connector says $SEND_STRING;
+														done
+													}
+													for i in $(seq $CONN_TRIES); do
+														if msg | nc -v $REMOTE_IP $REMOTE_PORT -w $CONN_TIMEOUT; then
+															break;
+														fi;
+													done >/dev/termination-log 2>&1`},
```

Comment:
This looks indented too far, at least in the github web UI.

Reply:
Ooops, right, let me handle this.